src/jit/codegencommon.cpp

```diff
@@ -5168,7 +5167,6 @@ void CodeGen::genPushCalleeSavedRegisters()
             offset += 2 * REGSIZE_BYTES;
         }
     }
-
```

Comment:
Thanks! This came from the previous branch (another PR for ETW_EBP_FRAMED fix) I branched this one from. Fixed it.